hyperspy/learn/mva.py

```diff
@@ -500,21 +523,18 @@ def blind_source_separation(self,
                                 **kwargs):
         """Blind source separation (BSS) on the result on the
         decomposition.
-
         Available algorithms: FastICA, JADE, CuBICA, and TDSEP.
-
         For lazy signal, the factors or loadings are computed to perfom the
         BSS.
-
         Parameters
```

Comment:
The blank newline before `Parameters` is important. Otherwise the API documentation generated by Sphinx loses the description of the first two arguments. I noticed this by the warnings generated when calling `make html` in the `hyperspy/docs` folder.